### DISCUSSION TITLE
[Announcement] Award session for LIA synthesis

### DISCUSSION THREAD

**Discussion by KihongHeo** [prosyslab-classroom/cs424-program-reasoning#264] [Announcements]
Hi,

I am happy to see your hard work on the previous homework. I would like to invite the top 3 students to give a presentation to share their experiences next Mon:
1. Hoseong Lee (이호성)
2. Minsol Park (박민솔)
3. Seohyun Lee (이서현)

For the top scorer, please prepare a 15-minute presentation and send me your slide before the class via email.
We will also have an award session. Everybody, please attend. See last year's [award sessions](https://photos.app.goo.gl/vNvRoKR72v8hkwPR6).

**Comment by HoseongLee** [comment]
Thank you professor. I just want to point out that my name is spelled Hoseong Lee, not Hosung Lee.

**Comment by KihongHeo** [comment]
Fixed. Thanks.
